test/integration/app.js

```diff
@@ -62,6 +61,35 @@ describe('app', () => {
           done();
         });
     });
+
+    it('should return an object containing the number of nearby and open services that were requested', (done) => {
+      const openResults = 2;
+      const nearbyResults = 5;
+
+      chai.request(app)
+        .get('/nearby')
+        .query({
+          latitude: coords.latitude,
+          longitude: coords.longitude,
+          'limits:results:open': openResults,
+          'limits:results:nearby': nearbyResults,
+        })
+        .end((err, res) => {
+          expect(err).to.equal(null);
+          expect(res).to.have.status(200);
+          // eslint-disable-next-line no-unused-expressions
+          expect(res).to.be.json;
+
+          const nearby = res.body.nearby;
+          const open = res.body.open;
+
+          expect(nearby).to.be.instanceof(Array);
+          expect(open).to.be.instanceof(Array);
+          expect(nearby.length).to.be.equal(nearbyResults);
+          expect(open.length).to.be.equal(openResults);
+          done();
+        });
+    });
   });
 
   describe('nearby error handling', () => {
```

Comment:
This is more of a general observation.

The structure of the tests that I write differs subtly from yours.

In the example here:
```
describe 'nearby error handling'
  it 'when the open limit is supplied and is not a number',
    expect ...
    expect ...
```
 
I would try to write as something like
```
describe 'nearby error handling'
  describe 'when the open limit is supplied and is not a number',
    it 'should return a json error message'
      expect ...
      expect ...
    it 'should specify the expected range'
      expect  ...
```

Although functionally the same the difference comes when considering the tests as documentation of the code. The output for the test run for the first example doesn't describe the code behaviour. I'd say that 'it' should be followed by 'should' to describe behaviour.

[Update: just spotted that the getPharmacies tests or structured using describe/it/should]

@st3v3nhunt @mcheung-nhs - any thoughts on this?